auth/server/src/main/java/com/seagates3/controller/UserPolicyController.java

```diff
@@ -111,6 +113,38 @@ ServerResponse detach() throws DataAccessException {
     return serverResponse;
   }
 
+ public
+  ServerResponse list() throws DataAccessException {
+    String pathPrefix = requestBody.get("PathPrefix");
+    String userName = requestBody.get("UserName");
+
+    LOGGER.info("List policies for user: " + userName);
+    ServerResponse serverResponse = null;
+
+    try {
+      User user = userDAO.find(requestor.getAccount().getName(), userName);
+      checkIfUserExists(user, userName);
+      Map<String, Object> dataMap = new HashMap<String, Object>();
+      if (pathPrefix != null) {
+        dataMap.put("pathPrefix", pathPrefix);
+      }
+      dataMap.put("accountName", user.getAccountName());
+      dataMap.put("policyIds", user.getPolicyIds());
+      List<Policy> policies = policyDAO.findByIds(dataMap);
+      serverResponse =
+          userPolicyResponseGenerator.generateAttachedUserPolicyListResponse(
+              policies);
+    }
+    catch (DataAccessException ex) {
+      serverResponse = userPolicyResponseGenerator.internalServerError();
+    }
+    catch (GuardClauseException grdClsEx) {
+      serverResponse = grdClsEx.getServerResponse();
+    }
+
+    return serverResponse;
+  }
+
  private
   void checkIfUserExists(User user,
```

Comment:
this function should return boolean and the caller should be taking a call to raise exception or take other actions.
THis looks to be a common utility function. Do we have a utility class where we can start putting this as a static methods and can reuse it at multiple places

Reply:
Will be taken care in a separate ticket EOS-26886.